desktop/src/cg/group4/server/DesktopIDResolver.java

```diff
@@ -1,4 +1,6 @@
-package cg.group4.client;
+package cg.group4.server;
+
```

Comment:
It was first in the Client module, but it does not belong there (since it is a DesktopIDResolver). I moved it to the proper module, but will make a package in there called client, since it is something involving the client. Also reflected this change in Android.